### ISSUE TITLE
python3 support

### ISSUE THREAD

**Issue by konsumer** [espes/Slave-in-the-Magic-Mirror#29]
If this project still works, I'd be happy to make a PR for it in python3, as 2 has been depracated.

**Comment by mstooks** [comment]
Did you ever decide to move forward with this project?

**Comment by kirk86** [comment]
@konsumer Do you have a PR for python3 I'd be happy to test it on and old ios device 9.3.5

**Comment by konsumer** [comment]
@kirk86  Nah. I dropped the project, and it was like a year ago I couldn't get it to work in python3, with no response.
